docs/support.rst

```diff
@@ -17,10 +17,9 @@ Good questions for Stack Overflow would be:
 Community Support
 -----------------
 
-Read the Docs is a community supported site,
-nobody is paid to handle readthedocs.org support.
-We are hoping to bring in enough money with our `Gold`_ program to change that,
-so please sign up if you are able.
+Read the Docs is a community- and ad-supported site.
```

Comment:
`community-` is a bit odd here. I think maybe reword it:

Read the Docs is supported by community contributions and advertising.